addons/html_editor/static/src/main/chatgpt/chatgpt_translate_dialog.xml

```diff
@@ -1,7 +1,7 @@
 <templates id="template" xml:space="preserve">
 
 <t t-name="html_editor.ChatGPTTranslateDialog">
-    <Dialog size="'lg'" title="'Translate with AI'">
+    <Dialog size="'lg'" title.translate="'Translate with AI'">
```

Comment:
`.translate` already assumes a static string as its value. You need to remove the inner quotes:
```suggestion
    <Dialog size="'lg'" title.translate="Translate with AI">
```